styleguide/src/Components/SidebarFixed/SidebarFixed.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react'
+import { Icon } from '../../Icons'
+
+const backgroundShadowEffect = [
+  '[background:linear-gradient(#FFFFFF_33%,rgba(255,255,255,0)),linear-gradient(rgba(255,255,255,0),#FFFFFF_66%)_0_100%,radial-gradient(farthest-side_at_50%_0,rgba(156,156,156,0.5),rgba(0,0,0,0)),radial-gradient(farthest-side_at_50%_100%,_rgba(156,156,156,0.5),rgba(0,0,0,0))_0_100%]',
+  '[background-size:100%_15px,100%_15px,100%_5px,100%_5px]',
+  '[background-attachment:local,local,scroll,scroll]',
+  '[background-repeat:no-repeat]',
+  '[background-color:#FFFFFF]',
+  '[&_.form-group_.bg-base-1]:bg-transparent',
+]
+
+export const SidebarFixedComponent: React.FC<SidebarFixedProps> = ({
+  children,
+  footerActions,
+  isOpen = false,
+  onClose,
+  helpLink,
+}) => {
+  const [sidebarIsOpen, setSidebarIsOpen] = useState(false)
+
+  useEffect(() => {
+    setSidebarIsOpen(isOpen)
+  }, [isOpen])
+
+  const handleRequestCloseFunc = () => {
+    setSidebarIsOpen(false)
+    onClose?.()
+  }
+
+  return (
+    <div
+      className={`sidebar-fixed fixed z-40 lg:z-20 right-0 bottom-0 w-full h-full transition-all overflow-x-hidden ${
+        sidebarIsOpen
+          ? 'pointer-events-auto opacity-100'
+          : 'opacity-0 pointer-events-none'
+      }`}
+    >
+      <div
+        className="sidebar-fixed-backdrop fixed inset-0 h-full w-full bg-black-alpha -z-1"
+        onClick={handleRequestCloseFunc}
+      />
+      <div
+        className={`sidebar-fixed-inner mt-10 sm:mt-0 top-0 bottom-0 right-0 absolute flex flex-col justify-between bg-base-1 transform transition-transform shadow h-auto w-full max-w-[410px] ml-16 rounded-t-lg sm:rounded-none ${
```

Comment:
```suggestion
        className={`sidebar-fixed-inner mt-10 sm:mt-0 top-0 bottom-0 right-0 absolute flex flex-col justify-between bg-base-1 transform transition-transform shadow h-auto w-full sm:max-w-[410px] sm:ml-16 rounded-t-lg sm:rounded-none ${
```

Ficou estranho no mobile
<img width="538" alt="image" src="https://user-images.githubusercontent.com/7097946/175657864-b7dcde3c-277e-484e-8b5e-7f6f62672281.png">